support-frontend/assets/pages/subscriptions-redemption/api.js

```diff
@@ -89,6 +89,7 @@ function buildRegularPaymentRequest(
   } = user;
 
   const product = {
+    productType: 'DigitalPack',
```

Comment:
`import { DigitalPack } from 'helpers/subscriptions';`
```suggestion
    productType: DigitalPack,
```